metadata.yaml

```diff
@@ -21,10 +21,8 @@ peers:
 provides:
   zookeeper:
     interface: zookeeper
-  metrics-endpoint:
-    interface: prometheus_scrape
-  grafana-dashboard:
-    interface: grafana_dashboard
```

Comment:
Hola Raúl!

Are you sure you want to remove the `metrics-endpoint` and `grafana-dashboard` relations? 

Perhaps it is part of the requirement for this PR, I don't know.

Reply:
Having the new relation I don't see a reason to keep them around. Would these relations be needed for anything that grafana-agent can't do?

Reply:
It is true that the grafana-agent relationship covers much of the functionality that `metrics-endpoint` and `grafana-dashboard` do.

There is a subtlety that is more of a product design decision than a technical decision:

By removing these relations your are removing the possibility that zookeeper can be **directly** scraped by any Prometheus and directly relate Grafana..... maybe there are some use cases where this is necessary. At this point you know more than I do